services/integration/src/models/constitution/constitution.models.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const constitutionSchema = new Schema({
+  sponsor: {
+    type: String,
+    required: [true, 'sponsor obligatory field']
+  },
+  customer: String,
+
+  assignment: Boolean,
+
+  contract: Boolean,
+
+  businessCase: Boolean,
+
+  utterance: Boolean,
+
+  strategicView: {
+    type: String,
+    required: [true, 'strategic view obligatory field']
+  },
+  descriptionProject: {
+    type: String,
+    required: [true, 'description obligatory field']
+  },
+  viabilityAnalysis: {
+    type: String,
+    required: [true, 'viability analysis obligatory field']
+  },
+  generalRequirements: {
+    type: String,
+    required: [true, 'general requirements obligatory field']
```

Comment:
These kind of  messages should be something like `'generalRequirements'` is a required field. This message is not well written. Also remember that a sentence starts with capitals.